metrics/lib/common.bash

```diff
@@ -21,6 +21,7 @@ KSM_BASE="/sys/kernel/mm/ksm"
 KSM_ENABLE_FILE="${KSM_BASE}/run"
 KSM_PAGES_FILE="${KSM_BASE}/pages_to_scan"
 KSM_SLEEP_FILE="${KSM_BASE}/sleep_millisecs"
+KSM_PAGES_SHARED="${KSM_BASE}/pages_shared"
```

Comment:
great - can you also then update any occurrences that are hard wired (by me 🙌 originally) in that same file - like in the ksm_settle func I reckon at least: https://github.com/kata-containers/tests/blob/master/metrics/lib/common.bash#L299